core/src/main/java/com/uber/m3/tally/HistogramImpl.java

```diff
@@ -130,23 +130,33 @@ ImmutableMap<String, String> getTags() {
     }
 
     private Duration getUpperBoundDurationForBucket(int bucketIndex) {
-        return bucketIndex < specification.getDurationUpperBounds().size() ? specification.getDurationUpperBounds().get(bucketIndex) : Duration.MAX_VALUE;
+        return bucketIndex < specification.getDurationUpperBounds().size()
+                ? specification.getDurationUpperBounds().get(bucketIndex)
+                : Duration.MAX_VALUE;
```

Comment:
for consistency - do you want to define Duration.POSITIVE_INF and so on? (i know its not needed for non-double comparison but might make the code easier to read, up to you)

Reply:
I had exactly the same thought when I started changing this code but it uses custom `Duration` type, which is just a wrapper on top of `long` primitive and Java doesn't have Infinity defined for `int` or `long`